pkg/retry/server.go

```diff
@@ -58,3 +67,10 @@ func (s *Server) handleRetry() http.Handler {
 		fmt.Fprint(w, `TODO: Make GitHub API calls and other stuff.. \n`)
 	})
 }
+
+func (s *Server) handleHealth() http.Handler {
```

Comment:
If you want a mini side project, we should move this (+ the grcp health) into the pkg/serving package to centralize it.

Reply:
ack

Reply:
merged https://github.com/abcxyz/pkg/pull/110